sdk/typescript/src/types/common.ts

```diff
@@ -13,6 +14,12 @@ import {
 import { CallArg } from './sui-bcs';
 import { fromB58 } from '@mysten/bcs';
 
+export const PaginationResponse = object({
```

Comment:
does this belong at the SDK level or app level?

Reply:
I went back and forth. I think either would really work. The reason i put it here is because we _could_ rework some schemas to just extend this so that we ensure we have uniform schemas here. But also we're going to eventually move this all go codegen so /shrug? I'll ask devx for their thoughts though.